JS/models/ExperimentalType.js

```diff
@@ -0,0 +1,8 @@
+const EXPEREMENTALTYPE = {
+  LIFTING_BODY: "Lifting_Body",
+  HYPERSONIC: "Hypersonic",
+  HIGH_ALTITUDE: "High_Altitude",
+  VTOL: "Vtol",
```

Comment:
What does this abbreviation mean? This could be unclear for those who are not familiar with the domain.